changelog/unreleased/issue-2571

```diff
@@ -0,0 +1,15 @@
+Bugfix: Check contents of unchanged files
```

Comment:
I would call this an enhancement, not a bugfix.

Reply:
I'm not sure. But changed it to Enhancement. IMO could be declared as both.. :wink: